meta/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
 galaxy_info:
-  author: "Anthony Ruhier"
-  description: Configures systemd-networkd profiles
+  role_name: systemd_networkd
```

Comment:
Note that this will change the name of the role in ansible galaxy from `aruhier.ansible_role_systemd_networkd` to `aruhier.systemd_networkd`.